project/Build.scala

```diff
@@ -1452,6 +1438,7 @@ object Build {
           name := "JavaLib Ex Test Suite",
           publishArtifact in Compile := false,
           testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a", "-s"),
+          scalacOptions += "-Ynowarn-default-junit-methods",
```

Comment:
OK that's good. It's still concerning, though. Any insight on why the plugin does not manage to set the option on its own?